integration/firestore/bundle_test_data/package.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
You should really be able to run this under ts-node or compile this once and check in the JS sources (as done in packages/firestore/scripts). Running this via ts-node will make your life much easier, as it makes debugging painless.

Reply:
Let's address this before other comments.

I tried to move it there and use tsc/node as other scripts, the issue I have is the generated js has 

`var bundle_data_1 = require("../test/unit/util/bundle_data");`

Which is expecting a bundle_data.js to exist, any idea how I can fix this? This is the first script to actually access the internals of the SDK, however, I think moving it here makes sense, and I want to make it happen now.

Reply:
To make this work in JS, you need to bundle all JS dependencies with the final script. Unfortunately, it looks like you may need a Rollup configuration for this after all. This will allow you define "externs" and decide which externs are inlined.

You can probably use your existing rollup configuration and put it in packages/firestore/rollup.script.js and then change https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore/package.json#L12 to invoke this instead.

Let me know if you need more help with this.

Reply:
Thanks for the help.